wpilibNewCommands/src/main/native/include/frc2/command/ParallelCommandGroup.h

```diff
@@ -74,13 +74,13 @@ class ParallelCommandGroup
     AddCommands(std::move(foo));
   }
 
-  void Initialize() override;
+  void Initialize() final;
 
-  void Execute() override;
+  void Execute() final;
 
-  void End(bool interrupted) override;
+  void End(bool interrupted) final;
 
-  bool IsFinished() override;
+  bool IsFinished() final;
```

Comment:
These should be marked `final override`. We are still overriding the base class method but don't want it to be overridden further.